tests/integration/activity.test.js

```diff
@@ -13,6 +22,126 @@ const test = async app => {
   const userUrl = urlparse(userId).path
   let activityUrl
 
+  // Create Reader object
+  const person = {
+    name: 'J. Random Reader',
+    authId: userId
+  }
```

Comment:
one thing I have been changing in my clean up is remove the authId from the person object. You don't need it, in your createReader, you pass in the userId anyways.